src/components/WeekWeatherData/WeekCard/index.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+import { weatherIconMap } from '../../../constants/iconMapping';
+import { toDateTime } from '../../../helper/functions';
```

Comment:
Look into webpack `alias` config, if you're interested
It can get rid of all the `../../../../../..`

Reply:
Yes we will create a separate task for that